omrmakefiles/rules.linux.mk

```diff
@@ -388,7 +409,11 @@ ifeq ($(OMR_OPTIMIZE),1)
                 ifeq (s390,$(OMR_HOST_ARCH))
                     OPTIMIZATION_FLAGS+=-O3 -mtune=z10 -march=z9-109 -mzarch
                 else
-                    OPTIMIZATION_FLAGS+=-O
+                    ifeq (riscv,$(OMR_HOST_ARCH))
+                      OPTIMIZATION_FLAGS+=-O3 -fno-strict-aliasing
+                    else
+                      OPTIMIZATION_FLAGS+=-O
+                    endif
```

Comment:
This can be written as `else ifeq (riscv,$(OMR_HOST_ARCH))` ... so no new nesting is required.

Reply:
Updated.